src/configure/configure.ts

```diff
@@ -124,46 +187,21 @@ class Orchestrator {
             await this.getAzureSession();
             await this.getGithubPatToken();
             await this.getSelectedPipeline();
+            if (!resourceNode) {
+                let selectedResource = await this.getAzureResource();
+                await this.selectTemplate(selectedResource, this.inputs);
+            }
+            else {
+                await this.selectTemplate(resourceNode, this.inputs);
+            }
 
-
-            if (this.inputs.pipelineConfiguration.templateInfo) {
-                let extendedPipelineTemplate = await templateHelper.getTemplateParameteres(this.inputs.azureSession, this.inputs.pipelineConfiguration.templateInfo);
+            if (this.inputs.pipelineConfiguration.template.templateType === TemplateType.remote) {
```

Comment:
Move this to a separate function so that getInputs() function is pretty easy to understand. And you can move the logic for populating inputs for AKS flow to else part here

Reply:
yes, I guess this is a part of a separate PR by @anuragc617